apps/test-netzer-encoder/src/main.cpp

```diff
@@ -0,0 +1,7 @@
+#include "NetzerEncoder.h"
+#include "mbed.h"
+
+int main(){
+	NetzerEncoder encoder(500000);
```

Comment:
that magic number should be a private static constexpr in the class
also no need to include mbed its included always